src/veins/base/utils/util.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <memory>
+#include <utility>
+
+/**
+ * User-defined implementation of std::make_unique.
+ *
+ * Until Veins builds on C++14 this provides equivalent functionality.
+ */
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args)
+{
+    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+}
```

Comment:
Suggestion: Add compile-time directives to automatically use C++14's `std::make_unique` if compiled with newer version.